archive/test/ArchiveControllerTest.scala

```diff
@@ -34,167 +34,167 @@ import services.RedirectService.{ArchiveRedirect, PermanentRedirect}
-    val path = "www.theguardian.com/books/worldliteraturetour/page/0,,2021886,.html"
+    val path = "/books/worldliteraturetour/page/0,,2021886,.html"
     val dest = "http://books.theguardian.com/worldliteraturetour/page/0,,2021886,.html"
     archiveController.linksToItself(path, dest) should be (true)
   }
 
   it should "lowercase the section of the url" in {
-    val result = archiveController.lookup("www.theguardian.com/Football/News_Story/0,1563,1655638,00.html")(TestRequest())
+    val result = archiveController.lookup("/Football/News_Story/0,1563,1655638,00.html")(TestRequest())
     status(result) should be (301)
-    location(result) should be ("http://www.theguardian.com/football/News_Story/0,1563,1655638,00.html")
+    location(result) should be ("/football/News_Story/0,1563,1655638,00.html")
   }
 
   it should "redirect century urls correctly" in {
-    val result = archiveController.lookup("www.theguardian.com/century")(TestRequest())
+    val result = archiveController.lookup("/century")(TestRequest())
     status(result) should be (301)
-    location(result) should be ("http://www.theguardian.com/world/2014/jul/31/-sp-how-the-guardian-covered-the-20th-century")
+    location(result) should be ("/world/2014/jul/31/-sp-how-the-guardian-covered-the-20th-century")
   }
 
   it should "redirect century decade urls correctly" in {
-    val result = archiveController.lookup("www.theguardian.com/1899-1909")(TestRequest())
+    val result = archiveController.lookup("/1899-1909")(TestRequest())
     status(result) should be (301)
-    location(result) should be ("http://www.theguardian.com/world/2014/jul/31/-sp-how-the-guardian-covered-the-20th-century")
+    location(result) should be ("/world/2014/jul/31/-sp-how-the-guardian-covered-the-20th-century")
   }
 
   it should "redirect an R1 century article to a corrected decade story endpoint" in {
-    val result = archiveController.lookup("www.theguardian.com/1899-1909/Story/0,,126404,00.html")(TestRequest())
+    val result = archiveController.lookup("/1899-1909/Story/0,,126404,00.html")(TestRequest())
     status(result) should be (301)
-    location(result) should be ("http://www.theguardian.com/century/1899-1909/Story/0,,126404,00.html")
+    location(result) should be ("/century/1899-1909/Story/0,,126404,00.html")
   }
 
   it should "not redirect a random URL that contains the word century" in {
-    val result = archiveController.lookup("www.theguardian.com/discover-culture/2014/jul/22/mid-century-textiles-then-and-now")(TestRequest())
+    val result = archiveController.lookup("/discover-culture/2014/jul/22/mid-century-textiles-then-and-now")(TestRequest())
     status(result) should be (404)
   }
 
-  it should "redirect failed combiners to the section" in {
-    val result = archiveController.lookup("www.theguardian.com/tv-and-radio/tvandradioblog+media/chris-evans")(TestRequest())
+  it should "redirect failed combiners to the (non-editionalised) section" in {
```

Comment:
this appears to have been intended to test a non-editionalised front, but tv-and-radio recently got editionalised